fetcher/tar.py

```diff
@@ -99,18 +101,26 @@ def unpack(self, downloadFirst=False):
         update = tarfile.open(name=self.workspacefile, mode='r:*')
         update.extractall(self.workspaceunpack)
```

Comment:
@floion the resinhup update packages (the archive) containes rootfs + boot + some
files. Extracting this in rootfs will most probably go over the rootfs size.